rust/sbp2json/src/bin/json2json.rs

```diff
@@ -12,10 +15,18 @@ static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
 ///
 /// Typical usage:
 ///
+///     json2json --input [PATH] --output [PATH]
```

Comment:
Need to update these example usage strings since we're not using option args (using positional args instead), example:

```
json2json [<INPUT> [<OUTPUT>]{
```

Reply:
What does --help show if we remove that completely? I feel like that's pretty similar to what clap will generate anyway

Reply:
Yeah, probably better to just remove this